packages/admin-api/lib/index.js

```diff
@@ -256,6 +256,29 @@ export default function GhostAdminAPI(options) {
             data,
             params,
             headers
+        }).catch((err) => {
+            /**
+             * @NOTE:
+             *
+             * If you are overriding `makeRequest`, we can't garantee that the returned format is the same, but
+             * we try to detect & return a proper error instance.
+             */
+            if (err.response && err.response.data && err.response.data.errors) {
```

Comment:
This is ok for launch IMO but need to prioritise doing this correctly as a shared tool. We need to get practiced at making new tools when we need to so that it doesn't feel like a huge thing.

Make request is going to want to be able to customise what this key is too I think.

Reply:
Agree 👍 . With errors it was a little different, because we already have a tool which handles errors, which we don't want to use atm (see disadvantages). Will raise an issue to re-think Ignition.